### DISCUSSION TITLE
Integrating SSO (SAML, Oauth2, OIDC etc.)

### DISCUSSION THREAD

**Discussion by kruscho** [apache/streampipes#2496] [Ideas]
Hi guys,

SSO would be a great option for easier user management (OIDC, OAuth2 or SAML). This would ideally be based/linked on this discussion [https://github.com/apache/streampipes/discussions/2400](https://github.com/apache/streampipes/discussions/2400)

**Comment by bossenti** [comment]
Hi @kruscho,

thank you very much for your feedback. I fully agree with you that supporting SSO would be a great addition to StreamPipes.
